subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -61,8 +61,8 @@ import static spock.lang.Retry.Mode.SETUP_FEATURE_CLEANUP
  */
 @ToolingApiTest
 @CleanupTestDirectory
-@ToolingApiVersion('>=3.0')
-@TargetGradleVersion('>=2.6')
+@ToolingApiVersion('>=7.0') // The lowest tested version should be the first release of the previous major.
```

Comment:
Does this mean we need to manually change it whenever we want to branch next major release `X.0`? If so, shall we add a step in release manual?

Reply:
Yes, that what I was thinking. Cany you please take care of it?